applications/FluidDynamicsApplication/custom_utilities/post_process_utilities.cpp

```diff
@@ -0,0 +1,47 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Miguel Angel Celigueta
+
+#include "post_process_utilities.h"
+#include "includes/define.h"
+#include "includes/variables.h"
+#include "includes/model_part.h"
+
+namespace Kratos {
+    typedef Node<3> NodeType;
+    typedef Geometry<NodeType> GeometryType;
+
+    void PostProcessUtilities::ComputeFlow(const ModelPart& rModelPart, double& flow) {
```

Comment:
```suggestion
    void PostProcessUtilities::ComputeFlow(
        const ModelPart& rModelPart,
        double& flow) {
```
Style convention for more than one argument methods.